MANIFEST-TEST.txt

```diff
@@ -10,16 +10,18 @@
     398861  2023-11-30T14:33:05Z  gs://dft-rlg-atip-test/layers/v1/local_authority_districts.geojson
   17063108  2023-11-30T14:33:07Z  gs://dft-rlg-atip-test/layers/v1/local_planning_authorities.pmtiles
    8939161  2023-11-30T14:33:08Z  gs://dft-rlg-atip-test/layers/v1/mrn.pmtiles
+  18806510  2024-01-23T15:59:25Z  gs://dft-rlg-atip-test/layers/v1/national_cycle_network.pmtiles
   21264435  2023-11-30T14:33:09Z  gs://dft-rlg-atip-test/layers/v1/parliamentary_constituencies.pmtiles
  107852920  2023-11-30T14:33:16Z  gs://dft-rlg-atip-test/layers/v1/pct_commute.pmtiles
   66994430  2023-11-30T14:33:20Z  gs://dft-rlg-atip-test/layers/v1/pct_school.pmtiles
     397024  2023-11-30T14:33:20Z  gs://dft-rlg-atip-test/layers/v1/railway_stations.geojson
   11569479  2023-11-30T14:33:21Z  gs://dft-rlg-atip-test/layers/v1/schools.pmtiles
   14056148  2023-11-30T14:33:23Z  gs://dft-rlg-atip-test/layers/v1/sports_spaces.pmtiles
+  23092043  2024-01-23T15:49:36Z  gs://dft-rlg-atip-test/layers/v1/stats19.pmtiles
    1067418  2023-11-30T14:33:23Z  gs://dft-rlg-atip-test/layers/v1/vehicle_counts.pmtiles
   30158538  2023-11-30T14:33:25Z  gs://dft-rlg-atip-test/layers/v1/wards.pmtiles
  403909863  2024-01-15T16:13:44Z  gs://dft-rlg-atip-test/private_layers/v1/pavement_widths.pmtiles
-  16312337  2024-01-12T14:08:39Z  gs://dft-rlg-atip-test/private_layers/v1/random_schemes.geojson
+  18043036  2024-01-23T16:10:09Z  gs://dft-rlg-atip-test/private_layers/v1/random_schemes.geojson
```

Comment:
We could also rename this now, but quite low priority, just an implementation detail